dev_guide/templates.adoc

```diff
@@ -122,8 +121,9 @@ uploading the template by piping the process output to the `create` command:
 ====
 
 ----
-$ osc process -f examples/sample-app/application-template-dockerbuild.json -v
-ADMIN_USERNAME=root,MYSQL_DATABASE=admin | osc create -f -
+$ osc process -f examples/sample-app/application-template-dockerbuild.json \
+    -v ADMIN_USERNAME=root,MYSQL_DATABASE=admin \
+    | osc create -f -
```

Comment:
I don't disagree. It's come up previously, but not captured in any guidelines. A question for Bilhar.